backends-common/opensearch/src/test/java/org/apache/james/backends/opensearch/DockerElasticSearchExtension.java

```diff
@@ -37,6 +38,7 @@ public void afterEach(ExtensionContext context) {
 
     @Override
     public void beforeEach(ExtensionContext extensionContext) {
+        elasticSearch.start();
```

Comment:
Ok didn't refresh well just saw this... Still not convinced though... it's in a beforeEach as well and you shut it down in afterAll... BeforeAll?

Reply:
I had an error before when putting it into BeforeAll. Not sure why but BTW this still does not work and likely not the problem...